### PR TITLE
Add iOS 12.2 and new iPads

### PR DESCRIPTION
**Pull request by denis-panov**
Added iOS 12.2 and new iPads to ControlCore configuration.

### CODE REVIEW

File: FBControlCore/Configuration/FBControlCoreConfigurationVariants.m [hunk 1]
```diff
@@ -31,9 +31,9 @@
 FBDeviceModel const FBDeviceModeliPhone8 = @"iPhone 8";
 FBDeviceModel const FBDeviceModeliPhone8Plus = @"iPhone 8 Plus";
 FBDeviceModel const FBDeviceModeliPhoneX = @"iPhone X";
-FBDeviceModel const FBDeviceModeliPhoneXS = @"iPhone XS";
-FBDeviceModel const FBDeviceModeliPhoneXSMax = @"iPhone XS Max";
-FBDeviceModel const FBDeviceModeliPhoneXR = @"iPhone XR";
+FBDeviceModel const FBDeviceModeliPhoneXS = @"iPhone Xs";
+FBDeviceModel const FBDeviceModeliPhoneXSMax = @"iPhone Xs Max";
+FBDeviceModel const FBDeviceModeliPhoneXR = @"iPhone Xʀ"; // U+0280 LATIN LETTER SMALL CAPITAL R
```
Comment (jmoody) [inline FBControlCore/Configuration/FBControlCoreConfigurationVariants.m:36]: 🤦‍♂️

File: FBControlCore/Configuration/FBControlCoreConfigurationVariants.m [hunk 2]
```diff
@@ -45,15 +45,20 @@
 FBDeviceModel const FBDeviceModeliPadPro_9_7_Inch_2ndGeneration = @"iPad Pro (9.7-inch) (2nd generation)";
 FBDeviceModel const FBDeviceModeliPadPro_12_9_Inch_2ndGeneration = @"iPad Pro (12.9-inch) (2nd generation)";
 FBDeviceModel const FBDeviceModeliPadPro_10_5_Inch = @"iPad Pro (10.5-inch)";
+FBDeviceModel const FBDeviceModeliPad_6thGeneration = @"iPad (6th generation)";
+FBDeviceModel const FBDeviceModeliPadPro_11_Inch = @"iPad Pro (11-inch)";
+FBDeviceModel const FBDeviceModeliPadPro_12_9_Inch_3ndGeneration = @"iPad Pro (12.9-inch) (3rd generation)";
```
Comment (jmoody) [inline FBControlCore/Configuration/FBControlCoreConfigurationVariants.m:50]: Great.

**Review by jmoody — APPROVED** [review]
Can you double check that we have all the correct model numbers?

Use this site: http://www.everyi.com/by-identifier/ipod-iphone-ipad-specs-by-model-identifier.html

Sometimes we miss Japanese and Chinese models.

If you are confident, then please self-merge.

Thanks!

**Comment by denis-panov** [comment]
> Can you double check that we have all the correct model numbers?
> 
> Use this site: http://www.everyi.com/by-identifier/ipod-iphone-ipad-specs-by-model-identifier.html
> 
> Sometimes we miss Japanese and Chinese models.
> 
> If you are confident, then please self-merge.
> 
> Thanks!


@jmoody I checked, but in this site iPhone11,4 is not described. I found this info here https://gist.github.com/adamawolf/3048717#file-apple_mobile_device_types-txt and also compared with FB original repo.

**Comment by jmoody** [comment]
> iPhone11,4 is an odd one. Apple never released it anywhere as far as anyone knows, but it's in the IPSW link anyway

We can keep the iPhone 11,4 , but for future reference we use the link I provided as ground-truth.

**Pull request merged**